Client/Application/LeanplumIntegration.swift

```diff
@@ -196,7 +217,14 @@ class LeanPlumClient {
             Leanplum.setAppId(settings.appId, withProductionKey: settings.productionKey)
             setupType = .production
         }
-
+        
+        // To be removed after beta testing
+        // Creating a beta user id for Leanplum testing as recommedded by LP Engineers
+        let userID = UUID().uuidString + "-Beta"
```

Comment:
Created a beta user id for Leanplum testing as recommended by LP Engineers. 

I can put a comment to remove it after we merge to beta or push another commit right after the beta build goes out?